t/17-labels_carry_over.t

```diff
@@ -61,12 +61,19 @@ subtest '"happy path": failed->failed carries over last issue reference' => sub
     };
 
     subtest 'carry over enabled in job group settings' => sub {
+        my $log = $t->app->log;
+        $t->app->log->level('debug')->unsubscribe('message');
+        my $messages = '';
+        my $cb = $log->on(message => sub { $messages .= pop . "\n" });
```

Comment:
This might have unintended side effects for later tests, since it modifies the app globally without reverting to the previous state.

Reply:
Wonder if i should make this an upstream test feature in Mojolicious itself, since it's so easy to mess up logging for the whole test. 🤔

Reply:
Currently the later tests seem fine.
Anyway, should I somehow resubscribe after that test?

Reply:
The problem is that the unsubscribe kills all future logging. So if a new test were to depend on it, things would get confusing for the next developer.

Reply:
I changed it to use `combined_from` now, but it would be good if we could avoid that in the future.